modules/command-engine/core/src/main/scala/surge/internal/persistence/PersistentActor.scala

```diff
@@ -211,10 +213,14 @@ class PersistentActor[S, M, R, E](
   private def freeToProcess(state: InternalActorState): Receive = {
     case pm: ProcessMessage[M] =>
       handle(state, pm)
-    case ae: ApplyEvent[E] => handle(state, ae)
-    case GetState(_)       => sender() ! StateResponse(state.stateOpt)
-    case ReceiveTimeout    => handlePassivate()
-    case Stop              => handleStop()
+    case ae: ApplyEvent[E] =>
```

Comment:
actually TBH I don't understand why we would ever need to handle this ApplyEvent message ? Since AggregateCommandModel can left fold over a number of events, why would we ever apply events one by one ...

Reply:
Cool, thanks for the explanation, I wasn't aware of the multi-DC aspect ;-)